src/utilities/byProgram.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+/**
+ * Generic helper methods to handle usage of MasterLists which
+ * are programs and program based requisitions and stock takes.
+ */
+
+/**
+ * Finds if the currently logged in store is a store which uses programs.
+ * If true then: there exists a master list which is a program, which has
+ * a store tag in programSettings which also exists in the tags for this
+ * store.
+ * @param  {Object}  settings
+ * @param  {Realm}   database
+ * @return {boolean} indicator of the currently logged in store using programs
+ */
+const thisStoreUsesPrograms = (settings, database) => {
+  const thisStoresTags = settings.get('ThisStoreTags');
+  const programs = database.objects('MasterList').filtered('isProgram = $0', true);
```

Comment:
also 'MasterList' that is joined to current store

Reply:
done b983e71ac72281f8329e9de43dd87dbdf1ce6aac